index.js

```diff
@@ -2,7 +2,11 @@ const express = require('express')
 const app = express()
 
 app.get('/', (req, res) => {
-  console.log('Hello World!')
+  res.send('Hello World!')
+})
+
+app.get('/new-path', (req, res) => {
+  res.send('This is a different path.')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/masterqwerty/sca-testing/pullRequest?prid=7713803)
```suggestion
  res.send("This is a different path.")
```